src/applications/form-renderer/tests/unit/utils/formConfig.unit.spec.js

```diff
@@ -6,11 +6,13 @@ import { expect } from 'chai';
 import sinon from 'sinon';
 import * as IntroductionPage from 'applications/form-renderer/containers/IntroductionPage';
 import { render } from '@testing-library/react';
+import * as webComponentPatterns from 'platform/forms-system/src/js/web-component-patterns';
+import * as addressPatterns from 'platform/forms-system/src/js/web-component-patterns/addressPattern';
```

Comment:
Sinon won't let me mock the address schema functions from the parent `web-component-patterns` export, so I need to import `addressPattern` separately.

Reply:
I'm curious about this. Can you say more about "won't let me"?

Reply:
Sinon doesn't recognize exports from `addressPattern` as being exports of `web-component-patterns`. I'm not entirely sure why, but it may not be following the import chain through to the root files. Everything else in the code seems to be able to import from `web-component-patterns` just fine, so this appears to be a Sinon bug, and may have been fixed in later versions.